src/main/java/org/prebid/server/bidder/openx/model/BidRequestWithError.java

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.bidder.openx.model;
+
+import com.iab.openrtb.request.BidRequest;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.util.List;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class BidRequestWithError {
```

Comment:
Check pls, m.b. we can avoid this class and pass errors list transitively along all logic for imps and requests processing.
This just because we don't have any dependency between bid requests and errors.

Reply:
Will do as  suggested, but for me it's seems less transparently to return BidRequest and as side action modifying the list passed as parameter by adding indicated errors.